<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>501</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>511</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>351</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>471</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Gambar 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Gambar Hasil Pemrosesan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Gambar 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>351</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1082</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuBuka">
     <property name="title">
      <string>Buka</string>
     </property>
     <addaction name="actionGambar_1"/>
     <addaction name="actionGambar_2"/>
    </widget>
    <addaction name="menuBuka"/>
    <addaction name="actionSimpan_Sebagai"/>
    <addaction name="actionKeluar"/>
   </widget>
   <widget class="QMenu" name="menuTentang">
    <property name="title">
     <string>Tentang</string>
    </property>
    <addaction name="actionTentang_Aplikasi"/>
   </widget>
   <widget class="QMenu" name="menuAritmetical_Operation">
    <property name="title">
     <string>Aritmetical Operation</string>
    </property>
    <addaction name="actionPenjumlahan"/>
    <addaction name="actionPengurangan"/>
    <addaction name="actionPerkalian"/>
    <addaction name="actionPembagian"/>
    <addaction name="actionAND"/>
    <addaction name="actionOR"/>
    <addaction name="actionXOR"/>
   </widget>
   <widget class="QMenu" name="menuClear">
    <property name="title">
     <string>Clear</string>
    </property>
    <addaction name="actionClearOutput"/>
    <addaction name="actionClearAll"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTentang"/>
   <addaction name="menuAritmetical_Operation"/>
   <addaction name="menuClear"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSimpan_Sebagai">
   <property name="text">
    <string>Simpan Sebagai....</string>
   </property>
  </action>
  <action name="actionKeluar">
   <property name="text">
    <string>Keluar</string>
   </property>
  </action>
  <action name="actionTentang_Aplikasi">
   <property name="text">
    <string>Tentang Aplikasi</string>
   </property>
  </action>
  <action name="actionPenjumlahan">
   <property name="text">
    <string>Penjumlahan</string>
   </property>
  </action>
  <action name="actionPengurangan">
   <property name="text">
    <string>Pengurangan</string>
   </property>
  </action>
  <action name="actionPerkalian">
   <property name="text">
    <string>Perkalian</string>
   </property>
  </action>
  <action name="actionPembagian">
   <property name="text">
    <string>Pembagian</string>
   </property>
  </action>
  <action name="actionAND">
   <property name="text">
    <string>AND</string>
   </property>
  </action>
  <action name="actionOR">
   <property name="text">
    <string>OR</string>
   </property>
  </action>
  <action name="actionXOR">
   <property name="text">
    <string>XOR</string>
   </property>
  </action>
  <action name="actionClearOutput">
   <property name="text">
    <string>Clear Output</string>
   </property>
  </action>
  <action name="actionClearAll">
   <property name="text">
    <string>Clear All</string>
   </property>
  </action>
  <action name="actionGambar_1">
   <property name="text">
    <string>Gambar 1</string>
   </property>
  </action>
  <action name="actionGambar_2">
   <property name="text">
    <string>Gambar 2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
